arm-operationsmanagement/2017-03-03-preview/Swagger/OperationsManagement.json

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
Could you please follow the guidelines outlined https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md#filenames-and-folder-structure .

Please make sure you create 3 commits. 
One with just api-version `2015-11-01-preview` 
Second with copy of `2015-11-01-preview` api-version into folder name `2017-03-03-preview`
Third with updating the only changes into the new api-version. We should only review the diffs between api-versions. 
Fourth with the adding [x-ms-examples](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md#why-x-ms-examples). 

 That's greatly help us in reviewing the changes. Thanks.